docs/dapp/emerald/writing-dapps-on-emerald.mdx

```diff
@@ -79,91 +79,9 @@ ParaTime.
 [Emerald Mainnet]: ./network.mdx
 [Emerald Testnet]: ./network.mdx
 
-## Running a Private Oasis Network Locally
+#### For development and testing, you can run a local [instance][localnet]
```

Comment:
```suggestion
## For development and testing, you can run a local [instance][localnet]
```
Probably 2 # too much
I think having the old header and card to the localnet could be also a good way

Reply:
Hmm. Yeah, `##` is more consistent with "Testnet" etc. Let me try the text for now. I tend to think that the card takes the reader away from the flow of the document.